src/main/assembly/dist/bag/metadata/files/2018/files.xsd

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (C) 2012 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:dcterms="http://purl.org/dc/terms/" xmlns:dc="http://purl.org/dc/elements/1.1/"
+    xmlns:f="http://easy.dans.knaw.nl/schemas/bag/metadata/files/" targetNamespace="http://easy.dans.knaw.nl/schemas/bag/metadata/files/"
+    elementFormDefault="qualified" attributeFormDefault="unqualified">
+
+    <!-- =================================================================================== -->
+    <xs:annotation>
+        <xs:documentation xml:lang="en"> This schema specifies a metadata-format for describing files and folders in a SIP. An instance of this
+            metadata-format can be used for ingest of SIPs with the Sword protocol. See also: https://easy.dans.knaw.nl/doc/sword2.html Created
+            2017-05-15 Copyright (c) 2017 DANS-KNAW Last update: 2018 - add additional-metadata element </xs:documentation>
```

Comment:
I like that you keep track of what happened to the schema. Let's keep doing that in the future for all our schemes!